docs/pages/build-reference/app-versions.mdx

```diff
@@ -34,11 +34,13 @@ When you are doing a production release, the user-facing version should be expli
 
 For developer-facing build version, you can set them to autoincrement on every build. This will help you avoid making manual changes to the project every time you upload a new archive on Play Store testing channels or TestFlight. One common cause for app store rejections is submitting a build with a duplicate version number. It happens when a developer forgets to increment the developer-facing build version number before creating a new build.
 
-EAS Build can help manage developer-facing build versions automatically by incrementing these versions for you if you opt into using the [`remote` version source](#remote-version-source). The default behavior is to use a `local` version source, which means you control versions manually in their respective config files.
+EAS Build can help manage developer-facing build versions automatically by incrementing these versions for you if you opt into using the [`remote` version source](#remote-version-source), which is the default behaviour. Optionally, you can choose to use a `local` app version source, which means you control versions manually in their respective config files.
```

Comment:
I believe that after multiple review rounds we decided that there is no default value (expect for nonInteractive), but the value needs to be specified explicitly

Reply:
Yes, you're right, the docs change reflect the previous state from before the last reviews